modules/gdscript/tests/scripts/analyzer/errors/lambda_shadowing_arg.out

```diff
@@ -0,0 +1,6 @@
+GDTEST_OK
+>> WARNING
+>> Line: 4
+>> SHADOWED_VARIABLE
+>> The local function parameter "shadow" is shadowing an already-declared variable at line 1.
+shadow
```

Comment:
This and the other tests for warnings should be in the `warnings` folder. I did notice there's no such folder under `analyzer` and all of those are in `parser/warnings` (not sure why). So either you can move to `parser/warnings` and we sort them out later or add these to a new `analyzer/warnings` folder.

Reply:
Done.